pages/index.js

```diff
@@ -1,7 +1,106 @@
-export default function HomePage() {
-	return (
-		<>
-			<h1>My Cosmetics Store</h1>
-		</>
-	);
+import styled from "styled-components";
+
+import { Btn } from "@/components/Button";
+
+import React, { useState } from "react";
+
+import myComponent from "@/components/myComponent1/myComponent.js";
+
+const kosmetikProdukt = [
```

Comment:
```suggestion
const cosmeticProduct = [
```

also, I would use an extra key-value pair for the currency and size values like: 

{
currency: "EUR",
measurement: "ml",
}

etc.